packages/common/src/types/ccdAmount.ts

```diff
@@ -22,7 +49,83 @@ export class CcdAmount {
         this.microCcdAmount = microCcdAmount;
     }
 
+    /**
+     * Returns the amount of micro CCD as a Big.
+     *
+     * @returns The amount of micro CCD as a Big
+     */
+    toBig(): Big {
```

Comment:
I think the name is a bit confusing, as `Big` could represent both CCD and microCCD amounts. I would prefer a name like
```suggestion
    toMicroCcd(): Big {
```
To align with the corresponding `toCcd` function for getting CCD value wrapped in `Big`